lib/mgmt_msg_native.h

```diff
@@ -233,16 +233,23 @@ _Static_assert(sizeof(struct mgmt_msg_tree_data) ==
 		       offsetof(struct mgmt_msg_tree_data, result),
 	       "Size mismatch");
 
+/* Flags for get-data request */
+#define GET_DATA_FLAG_STATE	0x01	/* get only "config false" data */
+#define GET_DATA_FLAG_CONFIG	0x02	/* get only "config true" data */
+#define GET_DATA_FLAG_ALL	(GET_DATA_FLAG_STATE | GET_DATA_FLAG_CONFIG)
```

Comment:
I almost put in my earlier comment, ditch `ALL`, should have... Please get rid of the `ALL` #define. It's just more clear what's going on in the code when one or both flags are present when need be. You are adding EXACT as a flag which is really incompatible with `ALL`. :)

Reply:
Removed.